.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+on:
+  push:
+    branches:
+    - '**'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.0'
```

Comment:
I initially had this as PHP 7.4, but the tests failed (created #69). I may try revising this to [use a matrix](https://github.com/marketplace/actions/setup-php-action#matrix-setup), but this seems good enough to start.

Reply:
Which reminds me that we need to upgrade to a more recent version of PHP _very_ soon.